core/trino-main/src/main/java/io/trino/operator/aggregation/state/Int128State.java

```diff
@@ -27,10 +27,4 @@
     long[] getDecimalArray();
```

Comment:
I would change the name here to like `getArray` as it is not only for decimal anymore

Reply:
Good point.